### ISSUE TITLE
Italian translation not working

### ISSUE THREAD

**Issue by zarik5** [JackD83/ALVR#286]
Reported on Discord. Requires more testing

**Comment by maxer456** [comment]
Tested some language settings to see what results in what display language. Since the i18n lib ALVR uses checks the browser's `navigator.language` (converted to lowercase), I'll focus on what I get there. The language value `it-it` (or `it-IT`) does give me the Italian translation.

Windows 10 display language "Italian" (any of the options):
The browser window ALVR.exe spawns respects Windows language settings.
With Edge it doesn't matter which Italian is selected in Windows, it always has just `it` as the value of `navigator.language`.
With Chrome I get `it-IT` no matter which Italian is selected in Windows.
Note that Edge was tested in a VM, Chrome on the system I usually use.

Browser settings:
I've added Firefox to the testing here since that's the browser I primarily use.
Firefox only has one Italian language option, whereas Chrome/Edge have 3.

browser (language) | value
--- | ---
Firefox (Italian) | `it-IT`
Chrome (Italian) | `it`
Chrome (Italian (Italy)) | `it-IT`
Chrome (Italian (Switzerland)) | `it-CH`

**Comment by JackD83** [comment]
Does this mean it works or not? 
The i18n is based on this plugin https://github.com/requirejs/i18n/

**Comment by zarik5** [comment]
Using "it-it" (lowercase) does work fine with me so I don't understand where is the problem. For me it works both on windows and on the headset. For the guy on Discord it would not work both on windows and on the headset.

**Comment by maxer456** [comment]
> Does this mean it works or not?
It works in cases where `navigator.language` is `it-IT`, which is ALVR using Chrome with Windows set to Italian, Chrome itself set to "Italian (Italy)" and Firefox set to Italian.

It does not work when the value is only `it`, like ALVR using Edge with Windows set to Italian.

Could the Italian translation be set for both `it` and `it-it` language codes? That would cover both cases for the ALVR browser window.

**Comment by JackD83** [comment]
Right now the language is implemented as it-it, this could be to specific. @maxer456 could you test if we used only "it"? Any browser set to it-it should fall back to use the "it" translation

**Comment by maxer456** [comment]
Tested with "it" in the language files instead of "it-it" and the translation now correctly shows up in all cases I tested before, so the fallback works fine.

I'll test this on the Quest inside ALVR later to confirm it works there too, but I don't doubt it will. Then I can push the fix.

Should the language code be changed for the German translation as well (from "de-de" to just "de")? I haven't tested these but since Chrome has multiple options for German available too, the same issue might show up.
